custom/succeed/models.py

```diff
@@ -107,4 +107,43 @@ class UCLAPatientFluff(fluff.IndicatorDocument):
 
     emitter = RandomizationDate()
 
+
+def get_full_name(case):
+    try:
+        return CommCareCase.get(case['indices'][0]['referenced_id'])['full_name']
+    except AttributeError:
+        return None
+
+
+class Emitter(fluff.Calculator):
+
+    @fluff.null_emitter
+    def date(self, case):
+        yield None
+
+
+class UCLATaskActivityFluff(fluff.IndicatorDocument):
```

Comment:
unfortunately not possible. I think you could add a custom expression to grab the first index id? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/README.md#extending-user-configurable-reports

I've been meaning to add a way to index into a list, but i haven't gotten around to it.